src/python/pants/backend/python/subsystems/repos.py

```diff
@@ -14,29 +15,53 @@ class PythonRepos(Subsystem):
         """
         External Python code repositories, such as PyPI.
 
-        These options may be used to point to custom cheeseshops when resolving requirements.
+        These options may be used to point to custom package indexes when resolving requirements.
         """
     )
 
     pypi_index = "https://pypi.org/simple/"
 
-    repos = StrListOption(
+    _find_links = StrListOption(
+        help=softwrap(
+            """
+            URLs and/or file paths corresponding to pip's `--find-links` option.
+
+            Per pip's documentations, URLs should be to HTML files with links to `.whl` and/or
```

Comment:
Just documentation instead of documentations I think.

Reply:
It also feels a bit mean in the way using #1234 is mean compared to using a full link that you may be able to click in your console or at least copy paste. Maybe worrying about dead links is a legitimate concern though. That said, there is also the risk of re-explaining the authoritative source and getting it wrong.